buildscripts/kokoro/linux_artifacts.sh

```diff
@@ -33,3 +33,14 @@ popd
 readonly MVN_ARTIFACT_DIR="${MVN_ARTIFACT_DIR:-$GRPC_JAVA_DIR/mvn-artifacts}"
 mkdir -p "$MVN_ARTIFACT_DIR"
 cp -r "$LOCAL_MVN_TEMP"/* "$MVN_ARTIFACT_DIR"/
+
+# for aarch64 platform
+apt-get install -y wget autoconf automake libtool g++-aarch64-linux-gnu g++ make
```

Comment:
wget, autoconf, automake, libtool, g++, and make should all already be available by the image supplied by Kokoro. Similarly openjdk and JAVA_HOME and really even update-ca-certificates seem like they shouldn't be necessary (we just ran java on the earlier lines of this file).

So I'd expect this entire thing to become something more like:
```sh
apt-get install -y g++-aarch64-linux-gnu
SKIP_TESTS=true ARCH=aarch_64 ./buildscripts/kokoro/unix.sh
```